packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	pressKeyWithModifier,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+async function focusBlockToolbar() {
+	// Move the mouse to show the block toolbar
+	await page.mouse.move( 0, 0 );
+	await page.mouse.move( 10, 10 );
+	await pressKeyWithModifier( 'shift', 'Tab' );
+}
+
+async function expectLabelToHaveFocus( label ) {
+	await expect(
+		await page.evaluate( () =>
+			document.activeElement.getAttribute( 'aria-label' )
+		)
+	).toBe( label );
+}
+
+async function testToolbar( currentBlockLabel ) {
```

Comment:
How about we rename those two test helpers to better express their purpose:

`testBlockToolbarKeyboardNavigation`
`testParentBlockKeyboardNavigation`

Plus another helper that wraps a block with a parent. It probably could be in the setup phase:
```js
await insert( 'Paragraph' );
await page.keyboard.type( 'Paragraph' );
await testBlockToolbarKeyboardNavigation( 'Paragraph block' );
await wrapCurrentBlockWithGroup();
await testParentBlockKeyboardNavigation( 'Paragraph block' );
```

What do you think? It might be too much, but on the other hand it took me some time to process this test.

Reply:
Thanks for the suggestions! I agree with you.

I would only suggest another name for the last function as its implementation is too specific for the Group block. Something like `testGroupKeyboardNavigation`.